### ISSUE TITLE
The tooltip for the toolbar icon controls (Refresh, Search, and Clear Lack Table) disappears when hovered over with the mouse: A11y_VSCode_Usability.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#286508]
### GitHubTags:
#A11yTCS;#DesktopApp; #Win32; #Visual Studio Code Client; #SH-VSCodeClient-Win32-April2025; #Keyboard; #A11yUsable;

### Environment and OS details:
Application Name: VS Code
Version: 1.108.0-insider (user setup)
Version 25H2
OS build 26200.7392

### Repro Steps:

1. Launch visual studio code Application.
2. Tab till copilot prompt chat section type a message and send.
3. Tab till refresh button.
4. Verify whether tooltip for the toolbar icon controls (e.g., "Refresh," "Search," and "Clear Lack Table") disappears when hovered over with the mouse.

### Actual Result:
The tooltip for the toolbar icon controls (e.g., "Refresh," "Search," and "Clear Lack Table") disappears when hovered over with the mouse.

### Expected Result:
The pointer should be able to move over the tooltip without it disappearing.
**Issue is  observed with Linux and mac also.**

### User Impact:
Users with motor impairments may struggle to move the pointer from the button to the tooltip without dismissing it prematurely.

### Attachment:

https://github.com/user-attachments/assets/42c0af0a-3e6f-4e40-8f01-c9bb2e0c561c

https://github.com/user-attachments/assets/38ef4cce-4893-4677-9418-b8f4f2ebdae2

MAC

https://github.com/user-attachments/assets/e25d4cd7-d600-4a7c-af3d-80c58e28c196

**Comment by benibenj** [comment]
This is normal and expected behaviour. Simple tooltips hide when hovering over them, this is the same with native browser tooltips. We do support pressing the "Alt" key to keep the tooltip visible in case the user want to select the text inside the tooltips
